src/d3a/models/strategy/load_hours.py

```diff
@@ -73,38 +85,45 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None, daily_budget=
             hrs_per_day = len(hrs_of_day)
         if hrs_of_day is None:
             hrs_of_day = list(range(24))
-
         self.hrs_of_day = hrs_of_day
         self._initial_hrs_per_day = hrs_per_day
-        self.balancing_energy_ratio = BalancingRatio(*balancing_energy_ratio)
 
         if not all([0 <= h <= 23 for h in hrs_of_day]):
             raise ValueError("Hrs_of_day list should contain integers between 0 and 23.")
 
         if len(hrs_of_day) < hrs_per_day:
             raise ValueError("Length of list 'hrs_of_day' must be greater equal 'hrs_per_day'")
 
-    @property
-    def active_markets(self):
```

Comment:
It is deleted here and then re-added above. Any particular reason for change of position